test/factories/compute_resources.rb

```diff
@@ -65,9 +65,9 @@
 
   factory :image do
     sequence(:name) { |n| "image#{n}" }
-    uuid Foreman.uuid
+    sequence(:uuid) { Foreman.uuid }
```

Comment:
`uuid { Foreman.uuid }` should also work. This isn't really a sequence, but a [dynamic attribute](http://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Dynamic_Attributes).

Reply:
Thanks, didn't know that. updated.